cloudsync/sync/manager.py

```diff
@@ -173,35 +175,46 @@ def __init__(self, state: SyncState,                        # pylint: disable=to
     def set_resolver(self, resolver):
         self._resolve_conflict = resolver
 
+    def _sync_one_entry(self, sync: SyncEntry):
+        something_got_done = False
+        try:
+            something_got_done = self.pre_sync(sync)
+            if not something_got_done:
+                something_got_done = self.sync(sync)
+            self.state.storage_commit()
+        except (ex.CloudTemporaryError, ex.CloudDisconnectedError, ex.CloudOutOfSpaceError, ex.CloudTokenError,
+                ex.CloudNamespaceError) as e:
+            log.warning(
+                "error %s[%s] while processing %s, %i", type(e), e, sync, sync.priority)
+            sync.punt()
+            self.__nmgr.notify_from_exception(SourceEnum.SYNC, e)
+            # do we want to self.state.storage_commit() here?
+            self.backoff()
```

Comment:
Backoff will raise a backoff error. This doesn't make sense if this is called synchronously outside the of a runnable do loop

Reply:
I realize this won't get called synchronously as smartsync overrides it. I am confused as to why the raise is in here but I see there's a comment next to it so it looks like you're on top of it

Reply:
Correct me if I'm wrong, but I think the plan is to not call this synchronously outside of a runnable do loop. If you want to `_sync_one_entry` then call the one in the SmartCloudSync class